internal/isoeditor/rhcos.go

```diff
@@ -382,6 +404,60 @@ func IgnitionImageArchive(ignitionConfig string) ([]byte, error) {
 	return compressedBuffer.Bytes(), nil
 }
 
+func readHeader(isoPath string) (*OffsetInfo, *OffsetInfo, error) {
+	iso, err := os.OpenFile(isoPath, os.O_RDONLY, 0o664)
+	if err != nil {
+		return nil, nil, err
+	}
+	defer iso.Close()
+
+	offsetMetadata := make([]byte, 48)
+	_, err = iso.ReadAt(offsetMetadata, headerLength-48)
+	if err != nil {
+		return nil, nil, err
+	}
```

Comment:
This doesn't look used, can it be removed?

Reply:
yep, just some leftovers.